modules/admin_manual/pages/qnap/index.adoc

```diff
@@ -0,0 +1,88 @@
+= ownCloud on a QNAP NAS
+
+== Introduction
+
+If you have a QNAP NAS (Network Attached Storage) system, you can install an ownCloud docker image from the QNAP App Center on your NAS. The free-of-charge community edition can be used with up to five users and an unlimited number of guest users.
+
+In case you want to use Enterprise features, you can purchase an Enterprise license from the QNAP store. This also allows you to purchase licenses for more users.
+
+The QNAP ownCloud is designed to work out of the box. We strongly recommend not to change the database (PostgreSQL) or the Web server (Apache). All configuration should happen in the Web interface. However command line configuration via `occ` commands is availabe.
+
+== Preparation
+
+If you haven't done so already, perform the following steps on you QNAP system:
+
+. Create a storage pool in addition to the system volume. You can combine hard disks and solid state disks in RAID groups for flexible storage. This pool can be resized later if necessary.
+
+. In the AppCenter, go to the QNAP Store and install Container Station version 2.0 or higher, which provides virtualization and Docker technogies. 
+
+. Enable `ssh` (secure shell) in the ControlPanel. In the navigation bar on the left side, select menu:Networks & File Services[Telnet / SSH]. Check `Allow SSH connection` and specify the port number (default 22). Next, check `Enable SFTP`. Once you click btn:[Apply], your admin user can log in to your NAS remotely.
```

Comment:
This is only needed I one wants to issue `occ` commands. So we should not require this in this step.

Reply:
Okay, moved to occ section for now.